ArduCopter/Parameters.cpp

```diff
@@ -1137,6 +1137,13 @@ const AP_Param::GroupInfo ParametersG2::var_info[] = {
     // @User: Standard
     AP_SUBGROUPINFO(command_model_pilot, "PILOT_Y_", 56, ParametersG2, AC_CommandModel),
 
+    // @Param: TKOFF_THRTIME
+    // @DisplayName: Ramp time of throttle during take-off
```

Comment:
Let's start this display name with Takeoff.

Reply:
Take-off throttle ramp time.

Reply:
How about TKOFF_SPOOL_TIME?

Reply:
Spool? To align with plane, but make it clear it's different (and using the actual wording from what Leonard had to say above) can I suggest TKOFF_SLEWTIME makes most sense. Then you have TKOFF_SLEWRATE for plane and TKOFF_SLEWTIME otherwise. Then when @tridge is convinced to change plane, the two parameters are right next to each other in the list.

Reply:
TKOFF_SLEWTIME is another decent option.  Here are the other similar parameters within Copter:

- MOT_SLEW_DN_TIME
- MOT_SLEW_UP_TIME
- MOT_SPOOL_TIME

Reply:
another option is MOT_THST_TIME.